components/budget/ExpenseBudgetItem.js

```diff
@@ -118,9 +118,11 @@ const ExpenseBudgetItem = ({
             {isLoading ? (
               <LoadingPlaceholder width={40} height={40} />
             ) : (
-              <StyledLink as={LinkCollective} collective={featuredProfile}>
-                <Avatar collective={featuredProfile} radius={40} />
-              </StyledLink>
+              <AvatarWithLink
+                radius={40}
```

Comment:
True, it is not a radius. `width` is a bit incomplete, because we're setting both the width and height. To be consistent with what we do in other places, I'm tempted to go with `size`. I'm gonna update here.